### ISSUE TITLE
client side console errors

### ISSUE THREAD

**Issue by nileio** [node-red/node-red#3326]
### Current Behavior

I am seeing the following errors in browser console repeatedly. 
Not sure if it it related to latest version of node-red or it is a `contib` node. If it is a `contrib` node that is not compatible with node-red 2.1 version, what will be the steps to find out which is the offending node?
Reporting to find out the root cause
Here is the console log
```
JQMIGRATE: Migrate is installed, version 3.3.0
Node-RED: 2.1.4

Versions
jQuery 3.5.1
jQuery UI 1.12.1
ACE 1.4.12
MONACO 0.28.1
D3 3.0.3

Definition error: gauth.label 
TypeError: Cannot read properties of undefined (reading 'creds') at Object.label (<anonymous>:8:52) at Object.getNodeLabel (http://localhost:1880/red/red.min.js?v=2.1.4:18:17188) at Object.createNodeIcon (http://localhost:1880/red/red.min.js?v=2.1.4:18:19867) at h (http://localhost:1880/red/red.min.js?v=2.1.4:18:305668) at j (http://localhost:1880/red/red.min.js?v=2.1.4:18:313720) at Object.emit (http://localhost:1880/red/red.min.js?v=2.1.4:16:20375) at pe (http://localhost:1880/red/red.min.js?v=2.1.4:16:56549) at Object.w [as import] (http://localhost:1880/red/red.min.js?v=2.1.4:16:76289) at Object.success (http://localhost:1880/red/red.min.js?v=2.1.4:16:4358) at c (http://localhost:1880/vendor/vendor.js?v=2.1.4:2:28294) {stack: "TypeError: Cannot read properties of undefined (re…/localhost:1880/vendor/vendor.js?v=2.1.4:2:28294)", message: "Cannot read properties of undefined (reading 'creds')"}
stack = "TypeError: Cannot read properties of undefined (reading 'creds')\n at Object.label (<anonymous>:8:52)\n at Object.getNodeLabel (http://localhost:1880/red/red.min.js?v=2.1.4:18:17188)\n at Object.createNodeIcon (http://localhost:1880/red/red.min.js?v=2.1.4:18:19867)\n at h (http://localhost:1880/red/red.min.js?v=2.1.4:18:305668)\n at j (http://localhost:1880/red/red.min.js?v=2.1.4:18:313720)\n at Object.emit (http://localhost:1880/red/red.min.js?v=2.1.4:16:20375)\n at pe (http://localhost:1880/red/red.min.js?v=2.1.4:16:56549)\n at Object.w [as import] (http://localhost:1880/red/red.min.js?v=2.1.4:16:76289)\n at Object.success (http://localhost:1880/red/red.min.js?v=2.1.4:16:4358)\n at c (http://localhost:1880/vendor/vendor.js?v=2.1.4:2:28294)"
message = "Cannot read properties of undefined (reading 'creds')"
[[Prototype]] = Error
constructor = function TypeError() { [native code] }
name = "TypeError"
message = ""
[[Prototype]] = Object
constructor = function Error() { [native code] }
name = "Error"
message = ""
toString = function toString() { [native code] }
length = 0
name = "toString"
[[Prototype]] = function () { [native code] }
[[Scopes]] = Scopes[0]
[[Prototype]] = Object
```

### Expected Behavior

no console errors

### Steps To Reproduce

noticed those only after upgrading to latest version

### Environment

- Node-RED version: 2.1.4
- Node.js version: 13.10.1
- npm version: 6.13.7
- Platform/OS: MacOS Catalina
- Browser: Chrome

**Comment by knolleary** [comment]
> `Definition error: gauth.label`

The error is in the label function of the `gauth` node type.

**Comment by nileio** [comment]
thanks. Yes found it it is the google-sheets node [node-red-contrib-google-sheets](https://flows.nodered.org/node/node-red-contrib-google-sheets)

**Comment by sammachin** [comment]
@nileio This is now fixed in v1.1.2 of node-red-contrib-google-sheets
